package-lock.json

```diff
@@ -96,96 +96,11 @@
         }
       }
     },
-    "@sentry/apm": {
```

Comment:
Why did `package-lock.json` change but `package.json` didn't? Had this just fallen behind?